spec/operators/concatAll-spec.ts

```diff
@@ -2,24 +2,33 @@ import { expect } from 'chai';
 import { from, throwError, of, Observable } from 'rxjs';
 import { concatAll, take, mergeMap } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
+import { observableMatcher } from 'spec/helpers/observableMatcher';
+import { expectObservable } from '../helpers/marble-testing';
 
 declare function asDiagram(arg: string): Function;
 declare const type: Function;
 declare const rxTestScheduler: TestScheduler;
 
 /** @test {concatAll} */
 describe('concatAll operator', () => {
+  let testScheduler: TestScheduler;
+
+  beforeEach(() => {
+    testScheduler = new TestScheduler(observableMatcher);
+  });
+
   asDiagram('concatAll')('should concat an observable of observables', () => {
-    const x = cold(    '----a------b------|                 ');
-    const y = cold(                      '---c-d---|        ');
-    const z = cold(                               '---e--f-|');
-    const outer = hot('-x---y----z------|              ', { x: x, y: y, z: z });
-    const expected =  '-----a------b---------c-d------e--f-|';
+    testScheduler.run(({ cold, hot, expectObservable }) => {
+      const x = cold(    '----a------b------|                 ');
+      const y = cold(                      '---c-d---|        ');
+      const z = cold(                               '---e--f-|');
+      const outer = hot('-x---y----z------|              ', { x: x, y: y, z: z });
```

Comment:
As with the other PR, all of the spaces used to align the marbles will need to be moved inside the string literals - so that reformatting with Prettier won't trash the alignment.